libs/designsystem/src/lib/components/item/item.component.scss

```diff
@@ -107,6 +115,12 @@
   }
 }
 
+:host(.disclosure) {
+  ion-item {
+    --inner-padding-end: 0px;
```

Comment:
Could we not just set this to `#{utils.size('xxs')}` and avoid setting padding-right on the icon?

Reply:
You're right! fixed